storage/integration_test.go

```diff
@@ -253,9 +253,11 @@ func testConfigGRPC(ctx context.Context, t *testing.T, opts ...option.ClientOpti
 
 // initTransportClients initializes Storage clients for each supported transport.
 func initTransportClients(ctx context.Context, t *testing.T, opts ...option.ClientOption) map[string]*Client {
+	withJSON := append(opts, WithJSONReads())
 	return map[string]*Client{
-		"http": testConfig(ctx, t, opts...),
-		"grpc": testConfigGRPC(ctx, t, opts...),
+		"http":      testConfig(ctx, t, opts...),
+		"grpc":      testConfigGRPC(ctx, t, opts...),
+		"jsonReads": testConfig(ctx, t, withJSON...),
```

Comment:
How much time does this add to the test run? Wondering if it's worth limiting this to reader tests rather than running all transports again.

Reply:
Tests are limited to reader tests using the `skipJSONReads` context.